homework9/src/pages/About/About.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import GoogleMapReact from "google-map-react";
+
+import ImageIcon from "../../components/ImageIcon/imageIcon";
+
+import mapIcon from "../../assets/img/map.svg";
+import phoneIcon from "../../assets/img/phone.svg";
+import fbIcon from "../../assets/img/facebook.svg";
+import instaIcon from "../../assets/img/instagram.svg";
+import twitterIcon from "../../assets/img/twitter.svg";
+
+import "./about.styles.css";
+
+const About = () => {
+  const renderMarkers = (map, maps) => {
+    let marker = new maps.Marker({
+      position: { lat: 39.826279, lng: -75.180313 },
+      map,
+      title: "Hello World!",
+    });
+    return marker;
```

Comment:
don't need to create separate variale just return  new maps.Marker(...

Reply:
Ok done